browser.js

```diff
@@ -17,7 +18,17 @@ module.exports = function(blob, cb){
           if (err) return cb(err);
 
           var buf = typedToBuffer(typed);
-          cb(null, parse(buf), buf);
+          var obj;
+
+          if (buf[0] === 60) {
+            obj = plistParse(buf.toString());
```

Comment:
can you tell me what you're referring to with "those" -- the zeroth-byte lookup? hit me up on slack if it's easier :-)